pom.xml

```diff
@@ -72,6 +72,17 @@
             <type>lutece-plugin</type>
             <version>[1.0.0,)</version>
         </dependency>
+        <dependency>
```

Comment:
This module already depends on the forms plugin, this could create circular dependency issues

Reply:
the unnecessary dependencies has been removed